src/main/resources/application.yml

```diff
@@ -54,6 +54,8 @@ oauth:
 
 identity:
   identityAPIUrl: "${oauth.serviceUrl}/api/identities"
+  mapForUidsUrl: "${oauth.serviceUrl}/map-for-uids"
+  removeReportingRolesUrl: "${identity.identityAPIUrl}/remove_reporting_roles"
```

Comment:
Can "/remove_reporting_roles" be changed to "/remove-reporting-roles" in identity-service if not impacting any other code.